docs/getting_started/installation/cpu/x86.inc.md

```diff
@@ -26,18 +24,39 @@ vLLM initially supports basic model inferencing and serving on x86 CPU platform,
 
 --8<-- "docs/getting_started/installation/cpu/build.inc.md"
 
-!!! note
-    - AVX512_BF16 is an extension ISA provides native BF16 data type conversion and vector product instructions, which brings some performance improvement compared with pure AVX512. The CPU backend build script will check the host CPU flags to determine whether to enable AVX512_BF16.
-    - If you want to force enable AVX512_BF16 for the cross-compilation, please set environment variable `VLLM_CPU_AVX512BF16=1` before the building.
```

Comment:
Is this no longer relevant?

Reply:
I think it's no need to expose these flags as they are rarely used. 

For cross-compilation and deploy to different targets, I recommend to use the image build arguments.

Reply:
Are these build arguments a user could use instead documented here?